### ISSUE TITLE
XML recording problems

### ISSUE THREAD

**Issue by judith-hash** [Blazemeter/CorrelationRecorder#17]
When I record xml code and I set 1 correlation rule WITH a Correlation Replacement all the XML code is record as:
<call-action><action name"Rule" rulename"RuleName"

Without the Correlation replacement, the XML code is record as:
<call-action><action name="Rule" rulename="RuleName"

and that is correct. So in the example the equal sign "=" was deleted. So is it a bug or do I sometimes wrong?

**Comment by RicardoPoleo** [comment]
Hello @judith-hash,

Thanks for taking the time to report this issue to us, we really appreciate it.

Based on the info you provided, looks like the Regular Expression you are using on the Corr. Replacement is also matching the sigh "=" inside the group and, when the replacement is done, removes that sign as well.

Until now, we haven't seen that behavior, so it isn't a known issue, but we can't dismiss it.

To give you a more accurate solution, we would need to know more info from you, for example:

1. Which version of the CorrelationRecorder are you using?
2. Which version of JMeter is this happening
3. Can you provide the configuration rules that you are using to do this recording? If you could share with us the Correlation Template/Test Plan/Definition of the Rules, that would be great to track it down.
4. It is possible that you share the URL and/or the JTL of the flow you are trying to Correlate? If sharing the JTL of the recording, please configure it as follows:

![JTLRecordingConfiguration](https://user-images.githubusercontent.com/7467545/102516861-1baf8500-406e-11eb-9152-b8fbb0abeb9f.png)

Please, let me know if you can share some of that info, so we can help you out,

And, as always, Happy Hacking!

**Comment by judith-hash** [comment]
Hello Ricardo,
Thanks for your quick response. See below the answers to your question.
1. The used version is version 1.1.1 of the Correlation Recorder Plugin.
2. It occurs on Jmeter 5.3 and 5.4.
3. The used Cor Rule is
Reference variable = functionsfingerprints_1
Corr. Extractor = Regex
  name="functions-fingerprints" content="\/api\/metadata\/Frontend.Functions\/fingerprints-(.+?)"
  1
  1
  BODY
  NOT Multivalued
Corr. Replacement = Regex
  fingerprints-(.+)
  ${functionsfingerprints_1}
  NOT ignore value
4. For now it is not possible to share the xml. See below recording code:
WRONG Recording
```xml
<call-action><action name"rule" rulename"GetCurrentAccountDataOnServer"><output bounded"no" local-name"MyOrganizationIID" remote-name"AccountOrganizationIID"/><output bounded"no" local-name"MainRoleType"/><output bounded"no" local-name"CurrentAccountIID"/><output bounded"no" local-name"CurrentParticipantIID"/></action></call-action>
```

RIGHT Recording
```xml
<call-action><action name="rule" rulename="GetCurrentAccountDataOnServer"><output bounded="no" local-name="MyOrganizationIID" remote-name="AccountOrganizationIID"/><output bounded="no" local-name="MainRoleType"/><output bounded="no" local-name="CurrentAccountIID"/><output bounded="no" local-name="CurrentParticipantIID"/></action></call-action>
```


WRONG Recording
```xml
<call-action><action name"rule" rulename"SMD.Common.GetCurrentUserCached"><output local-name"IsSuperUser" remote-name"issuperuser"/></action></call-action>
```

RIGHT Recording
```xml
<call-action><action name="rule" rulename="SMD.Common.GetCurrentUserCached"><output local-name="IsSuperUser" remote-name="issuperuser"/></action></call-action>
```

Regards, Jac

**Comment by judith-hash** [comment]
Hello Ricardo,
Do you have a possible solution on my comment?
Regards, Sjaak

**Comment by RicardoPoleo** [comment]
Hello @judith-hash,

This issue was fixed in version [v1.1.2](https://github.com/Blazemeter/CorrelationRecorder/releases/tag/v1.1.2) of the plugin. Please, check it out and hope this helps you.

I'll be closing this issue but, if you have any problem, please don't hesitate to re open it again.

Happy hacking,
